lib/ibanity/api/base_resource.rb

```diff
@@ -97,22 +97,31 @@ def prepare_attributes(raw)
 
     def setup_relationships(relationships, customer_access_token = nil)
       relationships.each do |key, relationship|
-        if relationship["data"]
-          self[Ibanity::Util.underscore("#{key}_id")] = relationship["data"]["id"]
-          return unless relationship.dig("links", "related")
-
-          klass = relationship_klass(key)
-          method_name = Ibanity::Util.underscore(key)
-          define_singleton_method(method_name) do |headers: nil|
-            klass.find_by_uri(uri: relationship["links"]["related"], headers: headers, customer_access_token: customer_access_token)
-          end
-        else
-          singular_key = key[0..-2]
-          klass        = relationship_klass(singular_key)
-          method_name  = Ibanity::Util.underscore(key)
-          define_singleton_method(method_name) do |headers: nil, **query_params|
-            uri = relationship["links"]["related"]
-            klass.list_by_uri(uri: uri, headers: headers, query_params: query_params, customer_access_token: customer_access_token)
+        url = relationship.dig("links", "related")
+        if url
```

Comment:
We don't always have a related link, so we should probably still expose the id to maintain backward compatibility
eg https://documentation.ibanity.com/ponto-connect/api#get-organization-usage

Reply:
We've already discussed that with Thibault and he said to forget about the relationship if it has no `links/related` element

Reply:
if i'm correct the relationship method was already not defined/not working without a related link, so i do not think we are breaking anyhting here, right?

Reply:
There was no method, but in the example I shared above there's no link to an organization endpoint, but we are currently setting an `organization_id` attribute from the relationship (see https://documentation.ibanity.com/ponto-connect/api/ruby#get-organization-usage). Not sure how useful it is, but we would be removing it with this change.

Reply:
I had to read the code 5 times but i finally get it :) 
Indeed we should continue to  define a <relationship_name>_id attribute in case we find something in relationship["data"]["id"]